std/net/curl.d

```diff
@@ -2122,6 +2134,9 @@ private mixin template Protocol()
         http.onReceive = (ubyte[] data) { return data.length; };
         http.setAuthentication("user", "pass");
         http.perform();
+
+        // Bugzilla 17540
```

Comment:
This is not a documented unittest, and referencing bugzilla numbers in tests is generally useful, so it's fine as it is.